src/containers/ConceptPage/EditConcept.tsx

```diff
@@ -6,58 +6,79 @@
  *
  */
 
-import React, { Fragment } from 'react';
-import PropTypes from 'prop-types';
+import React, { FC } from 'react';
 import { HelmetWithTracker } from '@ndla/tracker';
-import { injectT } from '@ndla/i18n';
+import { injectT, tType } from '@ndla/i18n';
 import ConceptForm from './components/ConceptForm';
 import { useFetchConceptData } from '../FormikForm/formikConceptHooks';
-import { LicensesArrayOf } from '../../shapes';
+import { License } from '../../interfaces';
+import { useTranslateConceptForm } from '../FormikForm/translateFormHooks';
+import Spinner from '../../components/Spinner';
 
```

Comment:
Du har gjort en bra jobb med å gjøre om .jsx til .tsx men tror vi kanskje burde vente med denne til vi gjør hele Concept pipelinen om til tsx. Det er vanskelig å teste om du har fått med alle prop'ene og om typen er riktig osv. når både 
øvre og nedre laget (ConceptPage og ConceptForm) er fortsatt skrevet i javascript. Tror det er større prosjekt å gjøre om Concept til typescript for vi må også tenke på hva vi gjør med redux og andre ting.